src/modulo1/tex/sections/teoremi_generali01.tex

```diff
@@ -68,7 +68,7 @@ \subsection{Teorema degli zeri}
         \centering
 	\begin{tikzpicture}
 	\begin{axis}[xmax = 4, xmin = -1.5, ymax = 2, ymin = -2,
-			     axis lines=middle, xlabel=$x$, ylabel=$y$,
+			axis lines=middle, xlabel=$x$, ylabel=$y$,
```

Comment:
Si è modificata l'indentazione di questa linea, farei in modo che torni come era prima